### ISSUE TITLE
How to make the tool work 

### ISSUE THREAD

**Issue by jeniffer007** [IBMa/equal-access#327]
When I run the command:

achecker C:\23_accessibility

I seem many failed. Could you please suggest why so many failed?

Here are the logs:

Passed: C:\23_accessibility\SSFC4F_2.3.0\about\architecture.html
Passed: C:\23_accessibility\SSFC4F_2.3.0\about\cloud_paks.html
Passed: C:\23_accessibility\SSFC4F_2.3.0\about\compliance.html
Failed: C:\23_accessibility\SSFC4F_2.3.0\about\cp_glossary.html
Failed: C:\23_accessibility\SSFC4F_2.3.0\about\cp_rh_version_compat.html
Passed: C:\23_accessibility\SSFC4F_2.3.0\about\cp_vids.html
Passed: C:\23_accessibility\SSFC4F_2.3.0\about\day0.html
Passed: C:\23_accessibility\SSFC4F_2.3.0\about\day1.html
Passed: C:\23_accessibility\SSFC4F_2.3.0\about\day2.html
Failed: C:\23_accessibility\SSFC4F_2.3.0\about\deprecate.html
Failed: C:\23_accessibility\SSFC4F_2.3.0\about\deprecated.html
Passed: C:\23_accessibility\SSFC4F_2.3.0\about\disable_net_isolation.html
Passed: C:\23_accessibility\SSFC4F_2.3.0\about\extend_update_support.html
Failed: C:\23_accessibility\SSFC4F_2.3.0\about\gdpr_readiness.html

net::ERR_FILE_NOT_FOUND at C:\23_accessibility\SSFC4F_2.3.0\nl\de\about\accessibility.html C:\23_accessibility\SSFC4F_2.3.0\nl\de\about\accessibility.html
net::ERR_FILE_NOT_FOUND at C:\23_accessibility\SSFC4F_2.3.0\nl\de\about\accessibility.html C:\23_accessibility\SSFC4F_2.3.0\nl\de\about\accessibility.html
(node:48160) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'aceBusy' of null
    at Function.<anonymous> (C:\Users\FengXiaXiao\AppData\Roaming\npm\node_modules\accessibility-checker\lib\ACBrowserManager.js:153:43)
    at step (C:\Users\FengXiaXiao\AppData\Roaming\npm\node_modules\accessibility-checker\lib\ACBrowserManager.js:33:23)
    at Object.next (C:\Users\FengXiaXiao\AppData\Roaming\npm\node_modules\accessibility-checker\lib\ACBrowserManager.js:14:53)
    at fulfilled (C:\Users\FengXiaXiao\AppData\Roaming\npm\node_modules\accessibility-checker\lib\ACBrowserManager.js:5:58)
(node:48160) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:48160) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by jeniffer007** [comment]
[log.txt](https://github.com/IBMa/equal-access/files/6066258/log.txt)

**Comment by tombrunet** [comment]
I think we discussed this on Slack where it's attempting to scan a file that doesn't exist. Is this issue now resolved?
